core/modules/wcontrol/TransmitMgr.h

```diff
@@ -62,51 +62,72 @@ class TransmitMgr {
     TransmitMgr& operator=(TransmitMgr const&) = delete;
     virtual ~TransmitMgr() = default;
 
-    int getTotalCount() { return _totalCount; }
-    int getTransmitCount() { return _transmitCount; }
-    int getAlreadyTransCount() { return _alreadyTransCount; }
+    int getTotalCount(int czarId);
```

Comment:
Normally the "get" methods are declared as `const`. If the only reason you don't do that is because of acquiring a lock onto the mutex then the standard approach is to declare the mutex as:
```
mutable std::mutex _mtx;
```
The `mutable' property would allow modifying the mutex's state from the `const` methods.